Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -592,6 +621,30 @@ function main () {
         }
     }
 
+    # if run with no flags and no account key, assume repair
+    if (!$repair -and !$reregister -and !$uninstall -and !$reinstall -and ($AccountKey -ne "__ACCOUNT KEY__")) {
+        LogMessage "No flags or account key found! Defaulting to the -repair flag."
+        $repair = $true
+    }
+
+    # if run with the repair flag, check if installed (install if not), if ver < 0.13.16 apply the fix
+    if ($repair) {
+        if (Test-Path(getAgentPath)){
+            $agentVersion = getAgentVersion
+            if ($agentVersion -notlike "0.13.16" -and $agentVersion -notlike "0.13.18") {
```

Comment:
@Alan-Huntress honestly, remove the agent versions here altogether. I think it's not going to hurt anything to have the repair script run on any version with the changes I'm asking for (don't stop the service, just start em)